ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+Playlist.swift

```diff
@@ -363,6 +363,45 @@ extension BrowserViewController: PlaylistScriptHandlerDelegate,
   ) {
     PlaylistP3A.recordUsage()
 
+    let addItemToPlaylist = {
+      [weak self] (
+        item: PlaylistInfo,
+        folderUUID: String?,
+        completion: ((_ didAddItem: Bool) -> Void)?
+      ) in
+      PlaylistItem.addItem(item, folderUUID: folderUUID, cachedData: nil) {
+        guard let self = self else { return }
+
+        if let url = URL(string: item.src), url.scheme == "blob" {
+          // Spawn a WebView to load the non-blob asset
+          Task { @MainActor in
+            let mediaStreamer = PlaylistMediaStreamer(
+              playerView: self.view,
+              webLoaderFactory: LivePlaylistWebLoaderFactory()
+            )
+
+            let newItem = (try? await mediaStreamer.loadMediaStreamingAsset(item)) ?? item
+            PlaylistManager.shared.autoDownload(item: newItem)
+
+            self.updatePlaylistURLBar(
+              tab: self.tabManager.selectedTab,
+              state: .existingItem,
+              item: newItem
+            )
```

Comment:
Is this needed?

Reply:
After a lot of testing, it's not needed, so I removed it. My main concern was that when I open the video from the page, it'd open the incorrect video because the URL is different, but I was wrong. It actually opens correctly and it deletes correctly.